snapshots/benchsuite/benchmark_test.go

```diff
@@ -54,7 +54,9 @@ func init() {
 	flag.StringVar(&nativeRootPath, "native.rootPath", "", "Root dir for native snapshotter")
 
 	// Avoid mixing benchmark output and INFO messages
-	logrus.SetLevel(logrus.ErrorLevel)
+	if err := log.SetLevel("error"); err != nil {
```

Comment:
Use log.ErrorLevel here?

Reply:
Right now `log.SetLevel` accepts a string not a `log.Level` or `logrus.Level`.

https://github.com/containerd/containerd/blob/03e4f1e3637ef7c0c33bdcb71642c02afa4f1298/log/context.go#L117-L125

And currently there are 2 patterns used to set log level using the `log` pkg:

1. `log.SetLevel("debug")` (string)
https://github.com/containerd/containerd/blob/03e4f1e3637ef7c0c33bdcb71642c02afa4f1298/cmd/containerd-stress/main.go#L180

2. `log.L.Logger.SetLevel(log.DebugLevel)` (const, but via the underlying `Logger`)
https://github.com/containerd/containerd/blob/03e4f1e3637ef7c0c33bdcb71642c02afa4f1298/runtime/v2/shim/shim.go#L170

I'm not sure which one is preferred, so I choose one of the two case by case based on which one is already used in the specific package.

I think since `log.SetLevel` just parses the string and call `L.Logger.SetLevel(lvl)`, maybe we should change its signature to `func SetLevel(level Level)` (basically `func SetLevel(level Level) { L.Logger.SetLevel(lvl) }`), and change all string usage to, e.g., `log.SetLevel(log.ErrorLevel)`? 🤔️

WDYT? @dmcgowan

Reply:
thinking more, the string argument seems to abstract `logrus.ParseLevel(level)` away from other packages when a level string is given (e.g., from cli flag). If so, maybe we can just use `log.SetLevel("level")` when there is user input, and `log.L.Logger.SetLevel(log.ErrorLevel)`? (a little verbose though)...